### ISSUE TITLE
[HOLD] User Avatar does not fall back on default when offline, and instead a blank space is shown.

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#5974]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Force close the app 
2. Turn on airplane mode
3. Open the app

## Expected Result:
App should load on offline mode and user avatar should be displayed 

## Actual Result:
App is not loading the user avatar on offline mode. A blank space is displayed. 

## Workaround:
Going back online fixes the issue. 

## Platform:

Where is this issue occurring?

- iOS
- Android
- Desktop

**Version Number:** 1.1.8-0

**Reproducible in staging?:** Yes 
**Reproducible in production?:** Yes 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

<img src="https://user-images.githubusercontent.com/44479856/138312427-2c4e6831-d3de-4f0e-856f-2b144d0da1b4.png" width="40%">

<img src="https://user-images.githubusercontent.com/44479856/138312441-b9681803-7c85-4559-b4a3-25f1731c7d54.png" width="40%">

When testing this, I noticed on my device that the avatars from the LHN conversation list were also not loading

<img src="https://user-images.githubusercontent.com/44479856/138312572-03dd1084-2e80-4101-a4c5-cdc90e1dd2c4.png" width="40%">

**Expensify/Expensify Issue URL:**

**Issue reported by:** @roryabraham 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1633893109079200

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

[view this job on upwork](https://www.upwork.com/jobs/~01382d9aca711ad10a)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @Luke9389 (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @jboniface (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by jboniface** [comment]
@Luke9389 do you have an opinion on the pending proposal?

**Comment by jboniface** [comment]
@developvsn if you haven't applied to the upwork job, can you do so? If you have applied, can you let me know who you are on UW so I can hire you? you can also message me through the job posting on UW and let me know your gh username if you'd prefer.

**Comment by developvsn** [comment]
Hello @Luke9389 , we started the PR (branch `fix-default-avatar`) !

Hello @jboniface, we'll message you on upwork , we haven't applied yet, thanks !

**Comment by mallenexpensify** [comment]
@developvsn we noticed the PR was submitted by @RGutu. Our process for managing and paying contributors is setup so that the same account that is hired in the Github issue is the same who submits the PR. Can you use the same accounts in the future?

**Comment by developvsn** [comment]
@mallenexpensify okay, we just wanted to submit it from our organisation account, and it didn't work, so in the future we'll use same accounts , understood.

**Comment by mallenexpensify** [comment]
Thanks @developvsn , it'll make our lives **much** easier.   

For this issue, can you apply to the Upwork job here https://www.upwork.com/jobs/~01382d9aca711ad10a then confirm in a comment here once you have.  Please include your first name and initial of last name so we can ensure we hire the correct person. 

I'm also going to confirm in the PR that payment will be going to the user in Upwork to ensure wires don't get crossed. 

Thanks!

**Comment by developvsn** [comment]
@mallenexpensify 
We applied to the Upwork job. The name on upwork is: Alexandru. S.

**Comment by mallenexpensify** [comment]
We are placing all non-critical issues on hold while we're on a company offsite.  The hold is expected to be lifted on 11/19 (but could go longer).  For any PRs that are submitted during the hold, we will add a $250 bonus payment.

**Comment by jboniface** [comment]
temporary hold, not overdue

**Comment by jboniface** [comment]
I hired Alexandru @developvsn

**Comment by MelvinBot** [comment]
📣 @developvsn You have been assigned to this job by @jboniface!
Please apply to this job in Upwork and let us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md) 📖

**Comment by developvsn** [comment]
@Luke9389 Hi ! Can you please review or give us an update regarding [Pull Request ](https://github.com/Expensify/App/pull/6209#issue-1044947049)


> @mallenexpensify okay, we just wanted to submit it from our organisation account, and it didn't work, so in the future we'll use same accounts , understood.

**Comment by mountiny** [comment]
@developvsn as per your comment [here](https://github.com/Expensify/App/issues/6386#issuecomment-975677994), I have tested this on Staging iOS app and the issue still prevails. Therefore, we should still go ahead with trying to solve this or find out why is it it should work on production but not on staging (I only have staging on my phone right now, so I can't test production behaviour).

**Comment by developvsn** [comment]
@mountiny @jboniface  @Luke9389  We downloaded production app for 3 different IOS devices and 1 android, and we can't reproduce this bag.

IOS

https://user-images.githubusercontent.com/71173662/142903856-b9380962-20ed-46e0-97a4-65e4118a7cba.MP4

Android

https://user-images.githubusercontent.com/71173662/142903920-75ea71b4-2b7e-4530-8463-7eb59f205f6f.mp4

We also tried to do it in github source code and it's the same result:


https://user-images.githubusercontent.com/71173662/142904731-cb82d08f-1f7b-446b-b3f3-46cb0cac9d39.mp4

**Comment by mountiny** [comment]
This is my experience on Staging. I guess we need to test more. Why would it occur only on staging but prod and dev would be fine, interesting 😅

https://user-images.githubusercontent.com/36083550/142930385-7ca83875-5a1c-476f-96c9-4823b42eafa8.MOV

**Comment by Luke9389** [comment]
For now, I'm going to put this issue on HOLD and then have this retested after the next deploy. 

Frankly, if this issue fixed itself, that'd be ideal, because the changes in the PR don't feel optimal to me. More discussion on that if need be. 

Rest assured @developvsn, we'll pay you for your time regardless of whether your PR is merged.

**Comment by Luke9389** [comment]
@isagoico Would you be willing to have this tested again? It looks like we had a successful deploy today, so we should have a clear idea of whether this is still occurring.

**Comment by isagoico** [comment]
Of course! happy to try and repro again :)

I was unable to reproduce this on Android on build 1.1.16-1. I currently don't have my iOS on me today so I'll check this tomorrow too (since it was the device I was able to reproduce initially).

**Comment by developvsn** [comment]
> Of course! happy to try and repro again :)
> 
> I was unable to reproduce this on Android on build 1.1.16-1. I currently don't have my iOS on me today so I'll check this tomorrow too (since it was the device I was able to reproduce initially).

Have you had a chance to look into it?

**Comment by isagoico** [comment]
I was unable to reproduce the Profile image issue on the main user but I was able to reproduce the missing profile picture in LHN conversation. Please check the video and image below 

https://user-images.githubusercontent.com/44479856/143454448-c2daf154-5ccd-4550-b915-a43a968e0e4b.MP4

![image](https://user-images.githubusercontent.com/44479856/143454633-201882e6-1438-4dfe-8eb7-d7a0c8ac1cf5.png)

**Comment by mountiny** [comment]
@isagoico It might be that the main user profile picture in your case is the default one, or at least I am thinking it might be a different scenario since originally Rory and then when I tried it, we both had a profile pic uploaded.

**Comment by developvsn** [comment]
Then what's the plan? Do we implement changes or keep it like this?

**Comment by mallenexpensify** [comment]
I just tested, it looks like MY avatar is showing but not all the rest.  FWIW...

<img width="380" alt="avatars" src="https://user-images.githubusercontent.com/22508304/144335526-64ab270b-6b82-414d-b49a-07baaeed6769.png">

**Comment by jboniface** [comment]
It looks like this is still being discussed?

**Comment by developvsn** [comment]
> It looks like this is still being discussed?

sort of. Waiting on Luke's [comment](https://github.com/Expensify/App/pull/6209#discussion_r760680905) here

It seems like it need more testing — looks like some devices loading avatars in offline and some not.

**Comment by Luke9389** [comment]
@jboniface, can we pay @developvsn for their time? 

I'd like to close this issue and open a new one with a more clear description of the problem and solution.

**Comment by developvsn** [comment]
> @jboniface, can we pay @developvsn for their time?
> 
> I'd like to close this issue and open a new one with a more clear (and up-to-date) description of the problem and solution.

Any updates? @jboniface

**Comment by jboniface** [comment]
@developvsn sorry, i missed that message. i just paid the contract. closing.
